client/components/fraud-risk-tools-banner/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useState } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Card } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { BannerBody, NewPill, BannerActions } from './components';
+import './style.scss';
+
+const FRTDiscoverabilityBanner = () => {
+	const { frtDiscoverBannerSettings } = wcpaySettings;
+	const { updateOptions } = useDispatch( 'wc/admin/options' );
+	const [ settings, setSettings ] = useState( () => {
+		try {
+			return JSON.parse( frtDiscoverBannerSettings );
+		} catch ( e ) {
+			return { remindMeCount: 0, remindMeAt: null, dontShowAgain: false };
+		}
+	} );
+
+	const showBanner =
+		( null === settings.remindMeAt && ! settings.dontShowAgain ) ||
+		( Date.now() > settings.remindMeAt && ! settings.dontShowAgain )
+			? true
+			: false;
+
+	const setReminder = () => {
+		const DAY_IN_MS = 24 * 60 * 60 * 1000;
```

Comment:
This same variable already exists on `client/overview/task-list/index.js`, so I'd suggest creating a `constants.ts` file at the client root (`client/constants.ts`) that can be imported and used for both files.

Reply:
Ok, I can do that. The only issue is that our TypeScript rules don't like the use of underscores:

```
Variable name `DAY_IN_MS` must match one of the following formats: camelCase, PascalCase
```

`DAYINMS` doesn't really read that well to me. Neither does `dayInMs`. Any variable name that you prefer to use here?

Reply:
I've implemented this in [3198f43](https://github.com/Automattic/woocommerce-payments/pull/5611/commits/3198f439f55fd25e710d2fb3007d13f8843f3203).